plugins/map/RedliningSupport.jsx

```diff
@@ -574,7 +575,7 @@ class RedliningSupport extends React.Component {
         feature.crs = this.props.mapCrs;
         const layer = {
             id: this.props.redlining.layer,
-            title: this.props.redlining.layerTitle,
+            title: LocaleUtils.tr(this.props.redlining.layerTitle),
```

Comment:
This line here is incorrect, layerTitle is already translated (see line 81 in Redlining.jsx).

Reply:
I belived too, but If I do not use it here It goes back to it's untranslated state as soon as you add a point in the layer.